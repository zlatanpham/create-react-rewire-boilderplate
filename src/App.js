import { hot } from 'react-hot-loader/root';
import React from 'react';
import Button from 'components/Button';

function App() {
  return (
    <div
      css={`
        padding-bottom: 50px;
        background-color: #ededed;
      `}
      className="flex items-center justify-center h-screen"
    >
      <Button>Click me please</Button>
    </div>
  );
}

export default hot(App);
