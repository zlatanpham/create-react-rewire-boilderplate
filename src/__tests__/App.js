import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

it('Should match snapshot', () => {
  const { container } = render(<App />);
  expect(container.firstChild).toMatchSnapshot();
});
