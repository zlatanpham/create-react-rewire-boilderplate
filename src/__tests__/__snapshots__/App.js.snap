// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
.c1 {
  background-color: #000;
  color: #fff;
  padding-left: 2.5rem;
  padding-right: 2.5rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
  border-radius: 9999px;
  font-size: 20px;
  font-weight: bold;
  border: none;
}

.c0 {
  padding-bottom: 50px;
  background-color: #ededed;
}

<div
  class="flex items-center justify-center h-screen c0"
>
  <button
    class="c1"
  >
    Click me please
  </button>
</div>
`;
